src/masternodes/govvariables/attributes.cpp

```diff
@@ -1533,6 +1535,39 @@ UniValue ATTRIBUTES::ExportFiltered(GovVarsFilter filter, const std::string &pre
+                    auto dvmDomain    = KeyBuilder(key, "dvm");
+                    auto evmDomain    = KeyBuilder(key, "evm");
+                    for (const auto &[id, value] : accounting->dvmEvmTotal.balances)
+                        ret.pushKV(KeyBuilder(dvmEvmEdge, id.v, "total"), ValueFromAmount(value));
+                    for (const auto &[id, value] : accounting->evmDvmTotal.balances)
+                        ret.pushKV(KeyBuilder(evmDvmEdge, id.v, "total"), ValueFromAmount(value));
+                    for (const auto &[id, value] : accounting->dvmCurrent.balances)
+                        ret.pushKV(KeyBuilder(dvmDomain, id.v, "current"), ValueFromAmount(value));
+                    for (const auto &[id, value] : accounting->dvmIn.balances)
+                        ret.pushKV(KeyBuilder(dvmDomain, id.v, "in"), ValueFromAmount(value));
+                    for (const auto &[id, value] : accounting->dvmOut.balances)
+                        ret.pushKV(KeyBuilder(dvmDomain, id.v, "out"), ValueFromAmount(value));
+                    for (const auto &[id, value] : accounting->evmCurrent.balances)
+                        ret.pushKV(KeyBuilder(evmDomain, id.v, "current"), ValueFromAmount(value));
+                    for (const auto &[id, value] : accounting->evmIn.balances)
+                        ret.pushKV(KeyBuilder(evmDomain, id.v, "in"), ValueFromAmount(value));
+                    for (const auto &[id, value] : accounting->evmOut.balances)
+                        ret.pushKV(KeyBuilder(evmDomain, id.v, "out"), ValueFromAmount(value));
+            } else if (const auto amounts = std::get_if<CEvmFees>(&attribute.second)) {
+                    auto evmFeesBlockKey     = KeyBuilder(key, "block");
+                    ret.pushKV(KeyBuilder(evmFeesBlockKey, "fee_burnt"), ValueFromAmount(amounts->feeBurnt));
+                    ret.pushKV(KeyBuilder(evmFeesBlockKey, "fee_burnt_min"), ValueFromAmount(amounts->feeBurntMin));
+                    ret.pushKV(KeyBuilder(evmFeesBlockKey, "fee_burnt_min_hash"), amounts->feeBurntMinHash.GetHex());
+                    ret.pushKV(KeyBuilder(evmFeesBlockKey, "fee_burnt_max"), ValueFromAmount(amounts->feeBurntMax));
+                    ret.pushKV(KeyBuilder(evmFeesBlockKey, "fee_burnt_max_hash"), amounts->feeBurntMaxHash.GetHex());
+                    ret.pushKV(KeyBuilder(evmFeesBlockKey, "fee_priority"), ValueFromAmount(amounts->feePriority));
+                    ret.pushKV(KeyBuilder(evmFeesBlockKey, "fee_priority_min"), ValueFromAmount(amounts->feePriorityMin));
+                    ret.pushKV(KeyBuilder(evmFeesBlockKey, "fee_priority_min_hash"), amounts->feePriorityMinHash.GetHex());
+                    ret.pushKV(KeyBuilder(evmFeesBlockKey, "fee_priority_max"), ValueFromAmount(amounts->feePriorityMax));
+                    ret.pushKV(KeyBuilder(evmFeesBlockKey, "fee_priority_max_hash"), amounts->feePriorityMaxHash.GetHex());
```

Comment:
Let's abstract all this into a simple array, that's looped and set. Clearer and less error prone.